frontier_service.py

```diff
@@ -85,6 +88,8 @@ def run(self) -> None:
         while True:
             self.fetch_peers()
             self.single_pass()
+            if not self.ready:
```

Comment:
I don;t understand what ready means in this case. It seems like a misleading thing.
It might not be as easy as I thought for the server to signal that it is ready.
Let's remove the ready flag and go back to sleeping for 3 seconds for now.
At least that one is much cleaner and easy to understand.